backend/functions/lib/adapters/gateways/pacer.api.gateway.ts

```diff
@@ -45,40 +48,42 @@ class PacerApiGateway implements PacerGatewayInterface {
       federalBankruptcyChapter: ['15'],
       dateFiledFrom: dateFileFrom,
     };
-    const response = await this.getCasesListFromPacerApi(body).catch((exception) => {
+    const response = await this.getCasesListFromPacerApi(body, context).catch((exception) => {
+      log.error(context, NAMESPACE, `PACER Case Locator API exception with ${exception.status} status: ${exception.message}`);
       throw new CaseLocatorException(exception.status, exception.message);
     });
 
     if (response.status != 200) {
+      log.error(context, NAMESPACE, `PACER Case Locator API returned ${response.status} response.`);
       throw new CaseLocatorException(response.status, 'Unexpected response from Pacer API');
     }
 
     return pacerToChapter15Data(response.data.content);
   }
 
-  public async getCasesListFromPacerApi(body: {}): Promise<HttpResponse> {
+  public async getCasesListFromPacerApi(body: {}, context: Context): Promise<HttpResponse> {
     const pacerCaseLocatorUrlBase = process.env.PACER_CASE_LOCATOR_URL;
     const pacerCaseLocatorUrlPath = '/pcl-public-api/rest/cases/find';
 
+    log.info(context, NAMESPACE, `Retrieving cases from PACER with the following request body: ${JSON.stringify(body)}`);
     return await httpPost({
       url: `${pacerCaseLocatorUrlBase}${pacerCaseLocatorUrlPath}`,
       headers: { 'X-NEXT-GEN-CSO': this.token },
       body,
     });
   }
 
-  public async getChapter15Cases(startingMonth?: number): Promise<Chapter15Case[]> {
+  public async getChapter15Cases(context: Context, startingMonth?: number): Promise<Chapter15Case[]> {
     if (startingMonth != undefined) {
       this.startingMonth = startingMonth;
```

Comment:
This is permanently overriding the default set in the constructor.  This probably has unintended side effects.